src/javaRosa.js

```diff
@@ -2102,12 +2102,12 @@ define([
                 {
                     name: "Edit Bulk Translations",
                     action: function (done) {
-                        _this.showItextDialog(done);
+                        _this.showItextModal(done);
                     }
                 }
             ]);
         },
-        showItextDialog: function (done) {
+        showItextModal: function (done) {
```

Comment:
Curious why you decided to rename from "Dialog" to "Modal". *Modal* is short for *modal dialog*, right?